### DISCUSSION TITLE
Support for container registry aliases within Helm-values.

### DISCUSSION THREAD

**Discussion by kad-hollac1** [renovatebot/renovate#23785] [Suggest an Idea]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.159.7

### If you're self-hosting Renovate, select which platform you are using.

Gitea or Forgejo

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

At our company we have container image definitions in our Helm values.
```
image:
  registry: '<alias>'
  repository: '<tenant>/envoy'
  tag: 'v1.26-latest'
```
However, Renovate is not picking up the alias since the config expected [here](https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/dockerfile/extract.ts#L168) is not passed in the helm-values manager, as can be seen [here](https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/helm-values/extract.ts#L21).

We defined the alias in the repositories `renovate.json` as follows.
```
  "registryAliases": {
    "<alias>": "notreal.azurecr.io"
  },
```
Reading the docs on `registryAliases`, we expected this to work, hence the bug report. However, one could also classify this as a feature request. The resulting url Renovate makes of the dependency is `https://registry-1.docker.io/v2/<alias>/<tenant>/envoy/tags/list?n=10000`.

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by viceice** [comment]
recategorizing as feature request
